pages/oss.md

```diff
@@ -23,7 +23,7 @@ As outlined in the OMB Open Government Directive,<sup id="fnr32"><a href="#fn32"
 
 In furtherance of the objectives outlined in the Open Government Directive, this policy requires that covered agencies participate in the following pilot program to encourage the development and publication of custom-developed Government code as OSS.
 
-Each covered agency shall release at least 20 percent of its newly-developed custom code each year as OSS. Custom code is defined as code for all custom software projects, modules, and add-ons that are self-contained.<sup id="fnr35"><a href="#fn35">35</a></sup> When deciding which custom code projects to release, each covered agency should prioritize the release of custom code that it considers potentially useful to the broader community.<sup id="fnr36"><a href="#fn36">36</a></sup>
+Each covered agency shall release at least 20 percent of its newly-developed custom code each year as OSS. Custom code is defined as code for all custom software projects, modules, and add-ons.<sup id="fnr35"><a href="#fn35">35</a></sup> When deciding which custom code projects to release, each covered agency should prioritize the release of custom code that it considers potentially useful to the broader community.<sup id="fnr36"><a href="#fn36">36</a></sup>
```

Comment:
The suggestion here is to remove the "self-contained" requirement of the pilot. "Self-contained" might be misinterpreted as meaning "must not integrate with any other (proprietary) software", which shouldn't be a reason for a piece of software to be kept private. It is common for a piece of software to integrate with a 3rd-party service (e.g. via an API), but that can be done in a way that does not infringe on any copyright.